README.md

```diff
@@ -77,6 +77,36 @@ pip install tensorflow
 
 In use cases where you require a small footprint, such as on a raspberry pi or similar embedded devices, you will want to install the TFLite Interpreter. You can install it for your platform by following [the instructions here](https://www.tensorflow.org/lite/guide/python#install_just_the_tensorflow_lite_interpreter).
 
+### Speech Pipeline
+
+The Speech Pipeline is the component that ties together VAD, Wakeword, and ASR. Essentially, this component listens to a frame of audio to determine if speech is present. If speech is detected, the Wakeword model processes the subsequent frames of audio looking for the specific keyword. If the keyword is found, the pipeline is activated and converts the following audio into a transcript. The Speech Pipeline is established like this from the previous initialized components:
```

Comment:
```suggestion
The Speech Pipeline is the module that ties together VAD, Wakeword, and ASR. The VAD (voice activity detection) listens to a frame of audio captured by the input device to determine if speech is present. If it is, the Wakeword model processes subsequent frames of audio looking for its keyword. If the keyword is found, the pipeline is activated and converts the following audio into a transcript. The Speech Pipeline is initialized like this:
```

Edited slightly for consistency with the terminology in the other platforms and because changing the order of the paragraphs changes what needs to be defined/introduced a bit.